app/components/ak-pagination-provider/index.ts

```diff
@@ -124,6 +126,11 @@ export default class AkPaginationProviderComponent<R> extends Component<
     );
   }
 
+  @action
+  updateTable() {
```

Comment:
rename this to `handleResultDependenciesChange` and dependency is passed on did-update